Source/Widgets/BaseLayerPicker/BaseLayerPicker.css

```diff
@@ -22,25 +25,32 @@
     border-radius: 10px;
     -moz-user-select: none;
     -webkit-user-select: none;
+    -ms-user-select: none;
     user-select: none;
     -webkit-transform: translate(0, -20%);
     -moz-transform: translate(0, -20%);
+    -ms-transform: translate(0, -20%);
+    -o-transform: translate(0, -20%);
     transform: translate(0, -20%);
     visibility: hidden;
     opacity: 0;
     -webkit-transition: visibility 0s 0.2s, opacity 0.2s ease-in, -webkit-transform 0.2s ease-in;
     -moz-transition: visibility 0s 0.2s, opacity 0.2s ease-in, -moz-transform 0.2s ease-in;
+    -o-transition: visibility 0s 0.2s, opacity 0.2s ease-in, transform 0.2s ease-in;
```

Comment:
Does Opera still use the -o- prefix now that they've switched to Blink?  I thought they ended up using the -webkit- prefix as a result.  Cesium was never compatible with Presto and the -o- prefix.